simulator.py

```diff
@@ -0,0 +1,123 @@
+# Class - Simulator
+# Runs multiple simulations of the game and saves the results
+
+import game
+import concurrent.futures
+import csv
+
+class Simulator:
+    """Class that manages simulations of a BlackJack game
+    
+    Parameters:
+
+    sim_count (int): Indicates the number of simulations to compute - will be rounded up to the
+    
+    nearest multiple of 100 
+    
+    bet (int): Represents the starting bet for the Black Jack game
+    """
+
+    def __init__(self, sim_count=1000000, bet_amount=1):
+        if sim_count % 10000 != 0: 
+            self.sim_count = sim_count + 10000 - (sim_count % 10000)
+        else:
+            self.sim_count = sim_count
```

Comment:
Do we have any plans to change the simulation count? If not I would prefer to hardcode it as a constant and remove this validation. It's a bit sloppy looking and unnecessary if we are the ones setting that value.

Reply:
I don't really see us changing the sim count during execution so I think we can just hardcode it like you said.

What do you think @esulu ?

Reply:
I think it would be good to allow for changing between longer/shorter runs, but then we'll also have to specify the increment rules if we accept input for instance